src/main/java/com/sqltest/demo/config/ScheduledTasks.java

```diff
@@ -0,0 +1,19 @@
+package com.sqltest.demo.config;
+
+import com.sqltest.demo.service.ExcelService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+@Component
+public class ScheduledTasks {
+    @Autowired
+    private ExcelService excelService;
+
+    @Scheduled(cron = "0 0 * * * ?") // Run every hour
```

Comment:
it will be costly if we run the schedulers in every hour, So define two times in a day. Specify a business time for FMS, Like in FMS application new intents are fetching only morning 10am and 4pm. Also add the cron value in application yaml and fetch the value in class.
eg: 
fms: 
  sync:
    cron: 0 7 0 * * ?

@Scheduled(cron = "${fms.sync.cron}")

Reply:
Update code:
fms.sync.cron=0 0 10,16 * * ?(At 10am, 4pm)